OpenEdXMobile/src/main/java/org/edx/mobile/util/AppConstants.java

```diff
@@ -11,6 +11,12 @@
     public static final String VIDEOLIST_BACK_PRESSED = "offline_video_back_pressed";
 
     public static final double MILLISECONDS_PER_SECOND = 1000.00;
+    // Threshold value to consider user has not rated yet
+    public static final float APP_NOT_RATED_THRESHOLD = 0.0f;
+    // Threshold value to consider user has given negative rate
+    public static final float APP_NEGATIVE_RATING_THRESHOLD = 3.0f;
+    // Minimum minor version changes required for negative raters to ask rate again.
```

Comment:
Minimum minor version changes required to ask negative raters to rate the app again